File: back-end/src/main/java/com/ta2khu75/quiz/entity/response/ExamResponse.java
package com.ta2khu75.quiz.entity.response;

import com.ta2khu75.quiz.entity.ExamLevel;
import com.ta2khu75.quiz.entity.ExamType;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
@Data
@NoArgsConstructor
@AllArgsConstructor
public class ExamResponse {
	Long id;
	String title;
	Integer time;
	String description;
	String imagePath;
	ExamType examType;
	ExamLevel examLevel;
}

File: front-end/src/model/response/ExamResponse.ts
import { AccessModifier } from "../AccessModifier";
import { ExamLevel } from "../ExamLevel";

export default interface ExamResponse{
    id: number;
    title:string;
    description:string;
    image_path:string,
    exam_level: ExamLevel,
    access_modifier: AccessModifier,
    exam_category:ExamCategoryResponse
}